### ISSUE TITLE
Sanitation strategy sce fails with fallback language

### ISSUE THREAD

**Issue by MarijnMensinga** [angular-translate/angular-translate#1618]
### Sanitation strategy `sce` fails with fallback language
When using a fallback language in conjuction with the sanitation strategy `sce` a type error is thrown when the string to translate is only available in the fallback language file.

Demo: https://plnkr.co/edit/iWdsq62cnGAY3mdN3aTM?p=preview

The `fallbackTranslationInstant` function expects a string as result but it gets a `TrustedValueHolderType` object.

```
typeError: result.substr is not a function
    at getFallbackTranslationInstant (angular-translate.js:1791)
    at resolveForFallbackLanguageInstant (angular-translate.js:1893)
    at fallbackTranslationInstant (angular-translate.js:1924)
    at determineTranslationInstant (angular-translate.js:2019)
    at Function.$translate.instant (angular-translate.js:2551)
    at translateFilter (angular-translate.js:3620)
    at fn (eval at compile (angular.js:14817), <anonymous>:4:97)
    at oneTimeInterceptedExpression (angular.js:16046)
    at oneTimeWatch (angular.js:15984)
    at Scope.$digest (angular.js:17515)
```

### Your environment
* version of angular-translate: 2.12.1
* version of angular: 1.5.8
* which browser and its version: Chrome 53.0.2785.143 m

### Steps to reproduce
- Two languages
- Set one langauge as fallback for another
- Set sanitation strategy to `sce`
- Translate a string via a filter

Demo: https://plnkr.co/edit/iWdsq62cnGAY3mdN3aTM?p=preview

### Expected behaviour
Everything should be nicely translated

### Actual behaviour
TypeError: `TrustedValueHolderType` object instead of expected `string`

**Comment by knalli** [comment]
Hm, did you have missed saving something in the demo? I guess, at least there is one missing file (angular-sanitize).

**Comment by MarijnMensinga** [comment]
Sorry the translation had to be done via filters in order to get the error, I updated the plnkr:
https://plnkr.co/edit/iWdsq62cnGAY3mdN3aTM?p=preview

**Comment by knalli** [comment]
Ok, demo working! 

Well, looks like sce does not working together with the linking feature (string based).

**Comment by MarijnMensinga** [comment]
@knalli are you planning on creating a fix for this? If so, when will it be released you think?

**Comment by knalli** [comment]
@MarijnMensinga It will take some days. If you can make a PR fixing the issue, it will faster.

**Comment by MarijnMensinga** [comment]
I've created a PR #1630 
This fixes my problems.. I'm not 100% sure this is enough for a permanent fix in the entire library?

**Comment by knalli** [comment]
Fix: aaaa972
